addon.xml

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="mediaimporter.plex" name="Plex Media Importer" version="0.0.1" provider-name="Montellese">
+<addon id="mediaimporter.plex" name="Plex Media Importer" version="0.0.2" provider-name="Montellese">
     <requires>
-        <import addon="xbmc.python" version="2.1.0"/>
+        <import addon="xbmc.python" version="2.26.0"/>
```

Comment:
This was introduced since you test builds are based on Matrix (api version is 3.0.0). 2.26.0 is the API version for Leia.